source/fundamentals/auth.txt

```diff
@@ -390,6 +390,7 @@ The next examples show how to provide your credentials by setting environment
 variables for the following types of authentication:
 
 - Programmatic access keys
+- Web identity provider
```

Comment:
Placed second in the list because it's checked immediately after programmatic access keys by the SDK